src/utils/index.ts

```diff
@@ -26,7 +26,12 @@ export const checkMissingDependencies = (dependencies: Array<string>): Array<str
         },
       });
       return false;
-    } catch (e) {
+    } catch (e: any) {
+      // If kubectl kustomize was ran and threw an exception that it could not find any kustomization file - should return false
+      if (e.message.includes("unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization'")) {
```

Comment:
I think you can remove this and run `kubectl kustomize --help` instead so you get an error only if it's not recognized

Reply:
Thanks for the advice!

Reply:
Done